content/en/blog/posts/2021-12-17-build-release-v0.7.0.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Shipwright v0.7.0 Is Here"
+date: 2021-12-17T10:30:00-04:00
+draft: false
+author: "Sascha Schwarze ([@SaschaSchwarze0](https://github.com/SaschaSchwarze0)) and Adam Kaplan ([@adambkaplan](https://github.com/adambkaplan))"
+---
+
+Ready for Christmas? We are, and our v0.7.0 release just made it!
+
+Is it a big thing just like our [previous v0.6.0 release](2021-10-26-build-release-v0.6.0.md)? No, but we have still done a couple of nice things:
+
+## Signed Images on ghcr.io
+
+We decided to go away from Quay as container registry for the images that we produce, and instead to consolidate things in GitHub. The new images can therefore be found in the [Packages section of our repository](https://github.com/orgs/shipwright-io/packages?repo_name=build). No worry, nothing that you need to bother so much. You can continue to install Shipwright through the Kubernetes manifest and that has the right image location in it.
+
+What else has changed with our images? We now sign them with [cosign](https://github.com/sigstore/cosign)! You can verify our controller image with the following command:
+
+```txt
+COSIGN_EXPERIMENTAL=1 cosign verify ghcr.io/shipwright-io/build/shipwright-build-controller:v0.7.0
+
+Verification for ghcr.io/shipwright-io/build/shipwright-build-controller:v0.7.0 --
+The following checks were performed on each of these signatures:
+  - The cosign claims were validated
+  - Existence of the claims in the transparency log was verified offline
+  - Any certificates were verified against the Fulcio roots.
+
+[{"critical":{"identity":{"docker-reference":"ghcr.io/shipwright-io/build/shipwright-build-controller"},"image":{"docker-manifest-digest":"sha256:887b76092d0e6f3c4f4c7b781589f41fde1c967ae9ae62f3a6bdbb18251a562f"},"type":"cosign container image signature"}...
+```
+
+Our signing process takes advantage of the new [keyless mode](https://github.com/sigstore/cosign/blob/main/KEYLESS.md) for cosign and support for [OIDC tokens in GitHub actions](https://chainguard.dev/posts/2021-12-01-zero-friction-keyless-signing).
+
+## shp - New `buildrun delete` Command
```

Comment:
this section is a lot more about code scans than the new shp buildrun delete 

I would redo the title along those lines and just lose the shp buildrun delete item